tests/Amazon.CloudWatch.EMF.Tests/Model/MetricDirectiveTests.cs

```diff
@@ -91,5 +91,17 @@ public void MetricDirective_PutMultipleDimensions_Returns_ValidJson()
 
             Assert.Equal("{\"Namespace\":\"aws-embedded-metrics\",\"Metrics\":[],\"Dimensions\":[[\"Region\",\"Instance\"]]}", jsonString);
         }
+
+        [Fact]
+        public void MetricDirective_PutMetricHighResolution_Returns_ValidJson()
+        {
+            MetricDirective metricDirective = new MetricDirective();
+            metricDirective.PutMetric("Time", 10,StorageResolution.HIGH);
+
+            string jsonString = JsonConvert.SerializeObject(metricDirective);
+
+            Assert.Equal("{\"Namespace\":\"aws-embedded-metrics\",\"Metrics\":[{\"Name\":\"Time\",\"Unit\":\"None\",\"StorageResolution\":\"1\"}],\"Dimensions\":[[]]}", jsonString);
+
+        }
```

Comment:
Can we also have one test to check if putMetric with explicitly specifying storage resolution as 'standard' returns valid JSON (storage resolution parameter is not present in the JSON).

`PutMetric("Time", 10, StorageResolution.STANDARD);`